ta4j-core/src/main/java/org/ta4j/core/BacktestExecutor.java

```diff
@@ -23,15 +23,15 @@
  */
 package org.ta4j.core;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import org.ta4j.core.cost.CostModel;
```

Comment:
unnecessary imports swapping

Reply:
It's common that java built in imports comes at first..and so on..

@team172011 even if you find such import swappings unnessecarry, it's not difficult to see that there is no trouble when merging..so you are asking me correct this issues? I will not, don't have time for such things..if you don't want to merge because of those simple import swappings then feel free to close this issue without merging and leave this bug as it is.